pkg/lib/cockpit-components-password.jsx

```diff
@@ -112,14 +125,6 @@ export const PasswordFormFields = ({
                     <div id={idPrefix + "-password-meter-message"} className="pf-c-form__helper-text" aria-live="polite">{passwordMessage}</div>
                 </div>
             </FormGroup>
-
-            {password_confirm_label && <FormGroup label={password_confirm_label}
```

Comment:
This commit is stronger than "do not require" -- to me it looks like "completely drop confirmation", which is reflected in the test changes. This seems rather dangerous to me -- typing a hidden password just once? There at least needs to be some big justification in the commit message, but I still don't quite like this. It's of course okay to not require confirmation *if* the pssword gets shown, but I don't see this conditional logic here?

Reply:
Indeed, it drops it. It follows up design from https://github.com/cockpit-project/cockpit/issues/16612#issuecomment-970163787

Reply:
The idea is that it only shows up when a weak password is made. It's a visual change (with new UI) with a warning color that isn't the default action (which would be disabled in this case).

Reply:
The old method is effectively double-clicking the same button, which is easy enough to do, especially for people who have a tremor (but also for _everyone_ accidentally misclicking once in a while), so having to select a secondary button in cases like these would be a better confirmation than clicking the same thing twice.

Reply:
@garrett we are talking here about dropping of "Confirm password" and adding he eye icon, not about the yellow button.

Reply:
Right, I was explaining how the button works in the context of a confirmation.

Reply:
Let's defer the second entry confirmation discussion and get the rest landed. The rest of the changes are well agreed upon; it would be a shame to not land them sooner than later.